sql/core/src/test/resources/sql-tests/inputs/describe.sql

```diff
@@ -97,3 +97,25 @@ DROP VIEW temp_v;
 DROP VIEW temp_Data_Source_View;
 
 DROP VIEW v;
+
+-- Show column default values
+CREATE TABLE d (a STRING DEFAULT 'show-create-table.sql', b INT DEFAULT 42) USING parquet COMMENT 'table_comment';
```

Comment:
Oh, I meant to refer `show-create-table.sql` file of this PR instead of mentioning the literal `show-create-table.sql`